### PR TITLE
fixed cautionary table on ipad

### PR DESCRIPTION
**Pull request by ksantiagoBAH**
## Description
https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/vets.gov-team/19580

On the profile page for a school with cautionary information, there is a table that breaks down the different types of complaints. This table does not show when viewing the page with an iPad.

The following screenshots are from the Trident Technical College profile page in production
https://www.va.gov/gi-bill-comparison-tool/profile/14915440

## Testing done
Local Testing

## Screenshots
Simulator Screen Shot - iPad (6th generation) - 2019-09-04 at 11 55 56

## Acceptance criteria
- [x] Table is displayed on iPad

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by zurbergram — APPROVED** [review]
LGTM

**Review by delli-gatti-michael — APPROVED** [review]
Approved by QA

File: src/applications/gi/components/profile/CautionaryInformation.jsx [hunk 1]
```diff
@@ -121,8 +122,12 @@ export class CautionaryInformation extends React.Component {
 
     const allComplaints = complaints.pop();
 
+    const cautionTableFix = environment.isProduction()
```
Comment (cvalarida) [inline src/applications/gi/components/profile/CautionaryInformation.jsx]: Why not push the fix out as soon as it's ready?

File: src/applications/gi/sass/partials/_gi-profile-page.scss [hunk 2]
```diff
@@ -163,6 +163,45 @@
       }
     }
   }
+  .cautionary-information-new {
```
Comment (cvalarida) [inline src/applications/gi/sass/partials/_gi-profile-page.scss]: I think you can apply the changes needed for this to work directly to `.cautionary-information`.

File: src/applications/gi/components/profile/CautionaryInformation.jsx (hunk 1 above)
Comment (ksantiagomorgan6) [reply src/applications/gi/components/profile/CautionaryInformation.jsx]: We want to test the fix in staging before the fix going out to production.

File: src/applications/gi/sass/partials/_gi-profile-page.scss (hunk 2 above)
Comment (ksantiagomorgan6) [reply src/applications/gi/sass/partials/_gi-profile-page.scss]: Once the prod flag is removed changes will be in `.cautionary-information`

**Review by cvalarida — APPROVED** [review]
Looks good. :+1: I appreciate the caution shown by testing in staging first. It's probably more than I would do for a CSS change, but it's definitely better to play it safe. My only ask is that it's all cleaned up after you're done testing. :wink:

**Pull request merged**